app/src/main/java/mozilla/org/webmaker/javascript/WebAppInterface.java

```diff
@@ -59,4 +66,10 @@ public void logText(String txt){
     public void setView(String url) {
         Router.sharedRouter().open(url);
     }
+
+    @JavascriptInterface
+    public String getRouteParams() {
+        Log.v("Router", mRoute.toString());
+        return mRoute.toString();
```

Comment:
hm, if you don't want to throw exceptions from JSON.parse, if it's possible this is `null`, you should alternatively return "{}".